bookwyrm/templates/snippets/status/status_header.html

```diff
@@ -1,72 +1,107 @@
 {% load bookwyrm_tags %}
 {% load i18n %}
-<span
-    itemprop="author"
-    itemscope
-    itemtype="https://schema.org/Person"
->
-    <a
-        href="{{ status.user.local_path }}"
-        itemprop="url"
-    >
-        {% include 'snippets/avatar.html' with user=status.user ariaHide="true" %}
+{% load humanize %}
 
-        <span itemprop="name">{{ status.user.display_name }}</span>
-    </a>
-</span>
+<div class="media">
+    <figure class="media-left" aria-hidden="true">
+        <a class="image is-48x48" href="{{ status.user.local_path }}">
+            {% include 'snippets/avatar.html' with user=status.user ariaHide="true" medium="true" %}
+        </a>    
+    </figure>
 
-{% if status.status_type == 'GeneratedNote' %}
-    {{ status.content | safe }}
-{% elif status.status_type == 'Rating' %}
-    {% trans "rated" %}
-{% elif status.status_type == 'Review' %}
-    {% trans "reviewed" %}
-{% elif status.status_type == 'Comment' %}
-    {% trans "commented on" %}
-{% elif status.status_type == 'Quotation' %}
-    {% trans "quoted" %}
-{% elif status.reply_parent %}
-    {% with parent_status=status|parent %}
+    <div class="media-content">
+        <h3 class="has-text-weight-bold">
+            <span
+                itemprop="author"
+                itemscope
+                itemtype="https://schema.org/Person"
+            >
+                {% if status.user.avatar %}
+                    <meta itemprop="image" content="/images/{{ status.user.avatar }}">
+                {% endif %}
 
-    {% if parent_status.status_type == 'Review' %}
-    {% blocktrans with username=parent_status.user.display_name user_path=parent_status.user.local_path status_path=parent_status.local_path %}replied to <a href="{{ user_path }}">{{ username}}'s</a> <a href="{{ status_path }}">review</a>{% endblocktrans %}
-    {% elif parent_status.status_type == 'Comment' %}
-    {% blocktrans with username=parent_status.user.display_name user_path=parent_status.user.local_path status_path=parent_status.local_path %}replied to <a href="{{ user_path }}">{{ username}}'s</a> <a href="{{ status_path }}">comment</a>{% endblocktrans %}
-    {% elif parent_status.status_type == 'Quotation' %}
-    {% blocktrans with username=parent_status.user.display_name user_path=parent_status.user.local_path status_path=parent_status.local_path %}replied to <a href="{{ user_path }}">{{ username}}'s</a> <a href="{{ status_path }}">quote</a>{% endblocktrans %}
-    {% else %}
-    {% blocktrans with username=parent_status.user.display_name user_path=parent_status.user.local_path status_path=parent_status.local_path %}replied to <a href="{{ user_path }}">{{ username}}'s</a> <a href="{{ status_path }}">status</a>{% endblocktrans %}
```

Comment:
It looks like this getting cut is why the tests aren't passing and replies don't render. If you add 
```html
               {% blocktrans with username=parent_status.user.display_name user_path=parent_status.user.local_path status_path=parent_status.local_path %}replied to <a href="{{ user_path }}">{{ username}}'s</a> <a href="{{ status_path }}">status</a>{% endblocktrans %}
               {% endwith %}
           {% endif %}
```

on line 43

and remove
```html
                        {% endwith %}
                    {% endif %}
```
on line 55 and 56, you'll be set.

You should be able to replicate the problem by loading a reply, or running the tests (`./bw-dev pytest`)

Reply:
Oooh yes thanks! I handled the merge from main and the conflicts resolution in GitHub and was now trying to troubleshoot it locally…